src/main/java/uk/gov/companieshouse/api/testdata/service/impl/TestDataServiceImpl.java

```diff
@@ -124,42 +124,79 @@ public CompanyData createCompanyData(final CompanySpec spec) throws DataExceptio
         final String companyNumberPrefix = spec.getJurisdiction().getCompanyNumberPrefix(spec);
 
         do {
-            // company number format: PP+123456 (Prefix either 0 or 2 chars, example uses 2 chars)
             spec.setCompanyNumber(companyNumberPrefix
                     + randomService
                     .getNumber(COMPANY_NUMBER_LENGTH - companyNumberPrefix.length()));
         } while (companyProfileService.companyExists(spec.getCompanyNumber()));
 
+        String currentService = null;
```

Comment:
currentService - can we take that out as it seems a bit overkill as the loggers will tell us where the code is